connector/src/main/java/org/geysermc/connector/network/translators/java/entity/JavaEntityStatusTranslator.java

```diff
@@ -49,6 +49,8 @@ public void translate(ServerEntityStatusPacket packet, GeyserSession session) {
         EntityEventPacket entityEventPacket = new EntityEventPacket();
         entityEventPacket.setRuntimeEntityId(entity.getGeyserId());
         switch (packet.getStatus()) {
+            case LIVING_DROWN:
+                entityEventPacket.setData(9);
```

Comment:
Needs a `break;` below this, and is there any sort of number constant in another class where you got 9 from?

Reply:
He said it’s what was sent by vanilla, so that’s what he used

Reply:
Yes, it's an intentional fall-though.